src/workers/job-processes.js

```diff
@@ -137,6 +137,7 @@ const processMap = {
 const syncProcessMap = {
   // 'failedMessageSender': failedMessageSender, //see method for danger
   handleIncomingMessages,
+  processSqsMessages,
```

Comment:
I just realized that as currently implemented, we'll process 10 messages per 5 minutes.  I think we'll need to add a process loop in job-processes that runs `await processSqsMessages()` inside a sleep loop the same way we do with some of the others. (like here: https://github.com/MoveOnOrg/Spoke/pull/310/files#diff-d82d53538123a15a45b9aa398d605c1aL28)  
but even as-is, this should still prove that it will actually load the data from sqs and process it anyway.